tests/data/oneaccess_lbb350.json

```diff
@@ -0,0 +1,2598 @@
+                    "ifInErrors_rate": null,
+                    "ifOutErrors": 0,
+                    "ifOutErrors_prev": 0,
+                    "ifOutErrors_delta": null,
+                    "ifOutErrors_rate": null,
+                    "ifInOctets": 656866288687,
+                    "ifInOctets_prev": 0,
+                    "ifInOctets_delta": null,
+                    "ifInOctets_rate": null,
+                    "ifOutOctets": 536260846008,
+                    "ifOutOctets_prev": 0,
+                    "ifOutOctets_delta": null,
+                    "ifOutOctets_rate": null,
+                    "poll_prev": null,
+                    "ifInNUcastPkts": 0,
+                    "ifInNUcastPkts_prev": 0,
+                    "ifInNUcastPkts_delta": null,
+                    "ifInNUcastPkts_rate": null,
+                    "ifOutNUcastPkts": 0,
+                    "ifOutNUcastPkts_prev": 0,
+                    "ifOutNUcastPkts_delta": null,
+                    "ifOutNUcastPkts_rate": null,
+                    "ifInDiscards": 0,
+                    "ifInDiscards_prev": 0,
+                    "ifInDiscards_delta": null,
+                    "ifInDiscards_rate": null,
+                    "ifOutDiscards": 37,
+                    "ifOutDiscards_prev": 0,
+                    "ifOutDiscards_delta": null,
+                    "ifOutDiscards_rate": null,
+                    "ifInUnknownProtos": 0,
+                    "ifInUnknownProtos_prev": 0,
+                    "ifInUnknownProtos_delta": null,
+                    "ifInUnknownProtos_rate": null,
+                    "ifInBroadcastPkts": 0,
+                    "ifInBroadcastPkts_prev": 0,
+                    "ifInBroadcastPkts_delta": null,
+                    "ifInBroadcastPkts_rate": null,
+                    "ifOutBroadcastPkts": 0,
+                    "ifOutBroadcastPkts_prev": 0,
+                    "ifOutBroadcastPkts_delta": null,
+                    "ifOutBroadcastPkts_rate": null,
+                    "ifInMulticastPkts": 0,
+                    "ifInMulticastPkts_prev": 0,
+                    "ifInMulticastPkts_delta": null,
+                    "ifInMulticastPkts_rate": null,
+                    "ifOutMulticastPkts": 0,
+                    "ifOutMulticastPkts_prev": 0,
+                    "ifOutMulticastPkts_delta": null,
+                    "ifOutMulticastPkts_rate": null
+                }
+            ]
+        }
+    },
+    "processors": {
+        "discovery": {
+            "processors": [
+                {
+                    "entPhysicalIndex": 0,
+                    "hrDeviceIndex": 0,
+                    "processor_oid": ".1.3.6.1.4.1.13191.10.3.3.1.2.3.1.3.0",
+                    "processor_index": "0",
+                    "processor_type": "oneaccess",
+                    "processor_usage": 71,
+                    "processor_descr": "Freescale T1040E (1.1) - Security Engine - 8-port Ethernet switc",
```

Comment:
Not really sure about the Processor name here. Seems that 4 CPUs are discovered. 1st one with this "Freescale ..." name, and the 3 others with just "Processor". If it is correct for the 1st one, may be you could find a way to have something a little bit more precise for the 3 others (so at least we can differentiate them). 

That being said, it looks very good.

Reply:
This Freescale is a 4 Core CPU
![image](https://user-images.githubusercontent.com/84004634/117922754-3a4f3a80-b2f3-11eb-9a75-283bbc14a1ca.png)
https://www.nxp.com/docs/en/fact-sheet/T1FAMILYFS.pdf
OneAccess use only one OID for the name of the CPU ...
"Description of the main CPU used on the main board"

Reply:
@PipoCanaja Hi, it's ok or not ?

Reply:
@mathieu-oui I started working on a fix for your processor description issue.  It was supposed to work, but the code that handles yaml was slightly wrong.

Reply:
i'm ready if you need some tests